### PR TITLE
Documentation update to reflect current status

### PR DESCRIPTION
**Pull request by eranmes**
Update the README.md file to reflect the fact that this library is fit for production use and is not just a sample.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,22 @@
-Android Key Attestation Sample
+Android Key Attestation Library
 ===================================
 
-This sample illustrates how to use the [Bouncy Castle ASN.1][1] parser to extract information
+This library uses the [Bouncy Castle ASN.1][1] parser to extract information
 from an Android attestation data structure to verify that a key pair has been
-generated in an Android device. 
+generated in an Android device. It is maintained in tandem with Android's key
```
Comment (sethmoo) [inline README.md:6]: Should we say "in Android Keystore" or something more specific?

Maybe "in a hardware-protected environment"?

File: README.md [hunk 2]
```diff
@@ -1,15 +1,22 @@
...
+attestation capabilities and is meant for production use.
 
-This repository contains a [server](server/) sample that shows how to attest an Android certificate
-outside the Android framework. This is the recommended best practice, since if the Android device
-is rooted or otherwise compromised on-device validation of the attestation may be inaccurate.
+This repository contains a [server](server/src/main/java/com/android/example/)
+sample code that shows how to validate an Android attestation certificate chain
+outside the Android framework. This is the recommended best practice, since if
+the Android device is rooted or otherwise compromised on-device validation of
```
Comment (sethmoo) [inline README.md:12]: nit: missing comma between "compromised" and "on-device".

**Review by sethmoo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (eranmes) [reply README.md:6]: Done.

File: README.md (hunk 2 above)
Comment (eranmes) [reply README.md]: Done.

Comment (eranmes) [reply README.md]: Done.

**Comment by sethmoo** [comment]
LGTM

**Comment by tnek** [comment]
I don't see, to have the ability to create issues in this repository so am commenting here:

Do you mind also publishing this server library into maven central? The current non-Google3 users I can find of it are all just directly [copying some version of it](https://github.com/search?q=com.google.android.attestation&type=code), which will eventually drift/be stale.

**Pull request merged**